testsuite/cluster-test/src/main.rs

```diff
@@ -302,8 +316,40 @@ impl BasicSwarmUtil {
             .iter()
             .map(|peer| parse_host_port(peer).unwrap())
             .collect();
+
+        let cluster = Cluster::from_host_port(parsed_peers, &args.mint_file);
+
+        let log_tail_started = Instant::now();
+        let logs = DebugPortLogThread::spawn_new(&cluster).0;
+        let log_tail_startup_time = Instant::now() - log_tail_started;
+        info!(
+            "Log tail thread started in {} ms",
+            log_tail_startup_time.as_millis()
+        );
+        let health_check_runner = HealthCheckRunner::new_all(cluster.clone());
+
         Self {
-            cluster: Cluster::from_host_port(parsed_peers, &args.mint_file),
+            logs,
+            cluster,
+            health_check_runner,
+        }
+    }
+
+    fn run_health_check(&mut self, duration: Duration) -> Result<()> {
```

Comment:
Any chance we can organize it similarly to `main.rs/emit_tx` function - have single function outside of any class(instead of duplication in BasicSwarmUtil and Runner)?